static/app/views/profiling/landing/table.tsx

```diff
@@ -0,0 +1,152 @@
+import {useMemo} from 'react';
```

Comment:
Similar naming Q here, landing/table is very generic. Can we rename this to TracesTable or similar? We need to be mindful about generic names like these because they pollute the global space when devs search around our repo.